app/views/products/_show.html.haml

```diff
@@ -22,5 +22,5 @@
     %h3 Documentation
     %ul.unstyled
       - docs.each do |stored_file|
-        %li.document= link_to stored_file.name, stored_file.file.url
+        %li.document= link_to stored_file.name, stored_file.download_url
```

Comment:
The tricky thing with that is also ensuring that permissions work properly. We don't want someone being able to change the URL by hand and get access to a file they shouldn't have access to.